.github/ISSUE_TEMPLATE/generic.md

```diff
@@ -3,3 +3,5 @@ name: Generic issue
 about: Blank template
 
 ---
+
+REMINDER: Does this issue also affect the user documentation? If so, add labels and create a matching issue in the [openj9-docs](https://github.com/eclipse-openj9/openj9-docs/issues) repo, per the [contribution guidelines](https://github.com/eclipse-openj9/openj9/blob/master/CONTRIBUTING.md).
```

Comment:
It would be helpful to specify which label(s) to add.

Reply:
I agree with @doveye, we want people to be reading the contribution guidelines, but I'm not sure that the issue template is the right place for that advice. The person raising the issue may not know whether the resolution will have a documentation component. Can this suggestion be injected into pull requests instead?

Reply:
I think PR-time might be too late, especially as the contribution guidelines say that ideally, the code PR shouldn't be merged until the doc PR is also ready. The contribution guidelines tell contributors to create a doc issue if required so they are already expected to know whether that's the case (they just need to consider whether any externals are affected). The guidelines say they can ask a committer to add a label so they can always ask if a docs issue is required if they're unsure.

Reply:
There's plenty of time between the creation of a pull request and when it's merged.

I think a documentation reminder (automatically inserted?) in a pull request would be helpful to committers as they consider merging a change.

Reply:
We could do both? Have a reminder in the issue and PR templates? Yet again for 0.33 we've had a doc issue created just a couple of working days before GA is due to start, for code updates that were done a month previously where it was obvious right from the start that doc changes would be needed. Doc issues should not be raised as an afterthought when someone comes to merge code updates.

Reply:
Is there such a thing as a PR template?

Reply:
See my last comment before this one.